templates/api/knowledge-panels/health/ingredients/ingredients_analysis_property.tt.json

```diff
@@ -1,4 +1,6 @@
 [% SET property = panel.property %]
+[% property_unknown_ingredients = property %]
+[% property_unknown_ingredients = property_unknown_ingredients.replace('-free', '-content-unknown') %]
```

Comment:
why not directly ?

```suggestion
[% # name of the unknown ingredients property %]
[% property_unknown_ingredients = property.replace('-free', '-content-unknown') %]
```

Reply:
You are right, but in fact while I was testing this change, I found that the feature did not output unknown ingredients when the property is otherwise known (e.g. if there's meat, we just display the meat, but not other unknown ingredients that might also be meat).

So I'm changing more things, and this logic will be removed.